### PR TITLE
Update pom file to include parent pom

### PR DESCRIPTION
**Pull request by mailming**
Please merge my change in, Thank you.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,14 +6,23 @@
 
 	<repositories>
 		<repository>
+			<id>turn</id>
+			<url>http://maven.turn.com/content/repositories/turn</url>
```
Comment (larry8lo) [inline pom.xml:10]: Is maven.turn.com visible from outside? Wouldn't this prevent non-Amobee people from using Camino?

Comment (mailming) [reply pom.xml:10]: Should we also make it available to public?

Comment (mailming) [reply pom.xml:10]: Our turn repo is not public.

Comment (mailming) [reply pom.xml:10]: This repo is only optional, it will look for main public repo if no access.

**Pull request merged**